drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -188,10 +189,30 @@ def self.apply_filters(input)
     Hmis::Filter::ProjectFilter.new(input).filter_scope(self)
   end
 
+  # Legacy referrals
   def receives_referrals?
     Hmis::Form::Instance.active.published.with_role(:REFERRAL).any? { |instance| instance.project_match(self) }
   end
 
+  def accepts_direct_ce_referrals?
+    config = Hmis::ProjectCeConfig.detect_best_config_for_project(self)
+
+    return false unless config.present?
+    return false unless config.accepts_direct_referrals?
+
+    true
```

Comment:
This could be simplified a bit
```suggestion
    config = Hmis::ProjectCeConfig.detect_best_config_for_project(self)
    config&.accepts_direct_referrals?
```